src/import/ImportWallet.tsx

```diff
@@ -274,15 +276,20 @@ const styles = StyleSheet.create({
     backgroundColor: colors.onboardingBackground,
   },
   scrollContainer: {
-    padding: 16,
+    paddingVertical: 16,
+    paddingHorizontal: 24,
   },
   button: {
-    paddingVertical: 16,
+    paddingVertical: 32,
   },
-  explanation: {
-    ...fontStyles.regular,
-    paddingHorizontal: 8,
+  title: {
+    textAlign: 'center',
+    ...fontStyles.h1,
+  },
+  description: {
     paddingTop: 16,
```

Comment:
nitya just updated this text to be center aligned and changed the padding on the bottom a bit after i asked about it :) so take a look at the designs again

Reply:
done